ethereum/rpc/namespaces/eth/api.go

```diff
@@ -1084,3 +1078,35 @@ func (e *PublicAPI) setTxDefaults(args rpctypes.SendTxArgs) (rpctypes.SendTxArgs
 
 	return args, nil
 }
+
+// getAccountNonce return the account nonce
+// Todo: include the ability to specify a blockNumber
+func getAccountNonce(ctx client.Context, backend backend.Backend, accAddr common.Address, pending bool) (uint64, error) {
+	_, nonce, err := ctx.AccountRetriever.GetAccountNumberSequence(ctx, accAddr.Bytes())
+	if err != nil {
+		return 0, err
+	}
+
+	if pending {
+		// the account retriever doesn't include the uncommitted transactions on the nonce so we need to
+		// to manually add them.
+		pendingTxs, err := backend.PendingTransactions()
+		if err != nil {
+			return 0, err
```

Comment:
I'd log the error here and return `nonce, nil`

Reply:
done